src/routes/recipeRouter.ts

```diff
@@ -1,7 +1,7 @@
 //Router for recipe endpoints

```

Comment:
you should remove comments like this one because everyone knows it is `Router for recipe endpoints`

because the name of this file point to it